src/main/java/org/folio/oaipmh/helpers/RepositoryConfigurationHelper.java

```diff
@@ -89,19 +85,14 @@
     return future;
   }
 
-  public static String getProperty(String name, Context ctx) {
-    return ctx.config().getString(name, System.getProperty(name));
-  }
-
   /**
    * Gets value of the config either from shared config or from System properties as a fallback.
    * @param tenant tenant
    * @param name config key
-   * @param ctx Vert.x context
    * @return value of the config either from shared config if present. Or from System properties as fallback.
    */
-  public static String getProperty(String tenant, String name, Context ctx) {
-    JsonObject configs = ctx.config().getJsonObject(tenant);
+  public static String getProperty(String tenant, String name) {
```

Comment:
The class seems to be a bit inconsistent - this method is declared as static, but another method is regular one, though both are utility methods.
I would propose the following changes:
1. Rename the class to RepositoryConfigurationUtil
2. Make both methods static
3. Rename the first method from getConfiguration to loadConfiguration since it does not return config